conf/citest/pipelines/LoadMembers.json

```diff
@@ -0,0 +1,63 @@
+{
+    "reference_db": "mysql://ensro@mysql-ens-compara-prod-6:4616/dthybert_citest_load_members_99",
+    "database_tests": {
+        "dnafrag": [
+            {   "test": "num_rows"
+            },
+            {   "test": "content" }
+        ],
+        "gene_member": [
+            {   "test": "num_rows"
+            },
+            {   "test": "content",
+                "columns": "-canonical_member_id"
```

Comment:
example.json would be a great place to explain the `-` syntax

Reply:
I'm afraid JSON does not support comments... I could add a README file, but for that you already have the docstring on the corresponding test, I guess.